web/ui/static/js/prom_console.js

```diff
@@ -332,6 +332,7 @@ PromConsole.Graph = function(params) {
 
   this.params = params;
   this.rendered_data = null;
+  this.rickshawGraph = null;
```

Comment:
Can you add a comment on this? Otherwise it's liable to be removed for being redundant.